src/rdplot/SimulationDataItemClasses/CsvLogs.py

```diff
@@ -46,13 +41,20 @@ def __init__(self, config, header, line):
 
         self.sequence = line[sequence_idx]
         self.qp = line[qp_idx]
-        self.config = config
+        self.config = dirname(normpath(path))
+
+        # use parsed info to set unique identifier for item
+        # not actually a real path like in the other simulation item classe!
+        self.path = path + f":{self.sequence}/{self.qp}"
 
         data = {}
         for i in range(0, len(header)):
             if i in [sequence_idx, qp_idx, rate_idx]:
                 continue
-            data[header[i]] = [(rate, float(line[i]))]
+            try:
```

Comment:
this protects against colums in the csv that are not for rdplot. for example exit status as a string